config/sync/language/da/core.date_format.long.yml

```diff
@@ -1,2 +1,2 @@
 label: 'Standard lang dato'
-pattern: 'l, j. F, Y - H:i'
+pattern: 'l, F j, Y - H:i'
```

Comment:
This is a slightly unconventional Danish date format. Is it a deliberate change?

Reply:
I agree. This would evaluate to "fredag, august 12, 2022 - 12:39", if I'm not mistaken. I cannot find an issue where we agreed to this format, so I will revert it.